src/components/sync-configurator/BaseSyncConfigurator.jsx

```diff
@@ -153,7 +242,7 @@ class BaseSyncConfigurator extends React.Component {
                         list={list}
                         customFiltersComponent={this.renderCustomFilters}
                         customFilters={filters}
-                        onSelectionChange={this.selectionChange}
```

Comment:
Following the previous idea, that would be: `onSelectionChange={this.changeSelection}`. Does it make sense?

Reply:
Indeed